src/sage/categories/finitely_generated_lambda_bracket_algebras.py

```diff
@@ -91,16 +92,17 @@ class Graded(GradedModulesCategory):
 
         EXAMPLES::
 
-            sage: LieConformalAlgebras(QQbar).FinitelyGenerated().Graded()
-            Category of H-graded finitely generated lie conformal algebras over Algebraic Field
+            sage: LieConformalAlgebras(QQbar).FinitelyGenerated().Graded()              # optional - sage.rings.number_field
+            Category of H-graded finitely generated lie conformal algebras
+             over Algebraic Field
         """
         def _repr_object_names(self):
             """
             The names of the objects of ``self``.
 
             EXAMPLES::
 
-                sage: LieConformalAlgebras(QQbar).WithBasis().FinitelyGenerated().Graded()
+                sage: LieConformalAlgebras(QQbar).WithBasis().FinitelyGenerated().Graded()                      # optional - sage.rings.number_field
```

Comment:
```suggestion
                sage: LieConformalAlgebras(QQbar).WithBasis().FinitelyGenerated().Graded()     # optional - sage.rings.number_field
```
Can do column 96 exceptionally if too long for 88.